pom.xml

```diff
@@ -46,20 +45,8 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
-                <configuration>
-                    <instructions>
-                        <Bundle-Activator>
-                            org.apache.sling.commons.threads.impl.Activator
-                        </Bundle-Activator>
-                        <Provide-Capability>
-                            osgi.service;objectClass=org.apache.sling.commons.threads.ThreadPoolManager,
-                            osgi.service;objectClass=org.apache.sling.commons.threads.jmx.ThreadPoolMBean
-                        </Provide-Capability>
-                    </instructions>
-                </configuration>
+                <groupId>biz.aQute.bnd</groupId>
```

Comment:
no need to duplicate this from parent: https://github.com/apache/sling-parent/blob/79c93908e42e6a3e09421c96ec2b96909741e891/sling-bundle-parent/pom.xml#L167. The relevant plugins are active by default.

Reply:
Fixed.